src/main/java/org/apache/mesos/hdfs/config/SchedulerConf.java

```diff
@@ -219,4 +219,12 @@ public int getReconciliationTimeout() {
   public int getDeadNodeTimeout() {
     return getConf().getInt("mesos.hdfs.deadnode.timeout.seconds", 90);
   }
+
+  public int getHealthCheckFrequency() {
+    return getConf().getInt("mesos.hdfs.healthcheck.frequency.ms", 60000);
```

Comment:
Why are these in ms while the other times are in seconds? Do you really expect a sub-second frequency/waiting period?